dvc/repo/plots/__init__.py

```diff
@@ -71,10 +72,21 @@ def collect(
 
                 if fs.isdir(path_info):
                     plot_files = []
-                    for pi in fs.walk_files(path_info):
-                        plot_files.append(
-                            (pi, relpath(pi, self.repo.root_dir))
+                    try:
+                        for pi in fs.walk_files(path_info):
+                            plot_files.append(
+                                (pi, relpath(pi, self.repo.root_dir))
+                            )
+                    except NoRemoteError:
```

Comment:
Same as we handle it in `dvc diff` https://github.com/iterative/dvc/blob/77be01ee2d8bed167928e64ab6d29c6be87554d0/dvc/repo/diff.py#L168

Reply:
Though alternative these days could be to use `onerror` too, but in this case we know that this dir is dvc-tracked, so this error could only be risen for the dir itself.

Reply:
(just trying to understand this better)

> Also NoRemoteError could be replaced with some standard errno

yep 👍 - it can potentially raise other things here? like remote is not available, or remote exists, but broken, right? 

not sure if we handle this well on the Studio side cc @Suor ... I mean the long tail of all other possible errors here ?

seems related to what @pared has been working on also (to not fail on errors on specific revisions), looks like it should be all part of that solution, wdyt guys?